docs/index.md

```diff
@@ -18,12 +18,21 @@ description: HyperFormula® - Complex Calculations Made Easy
 
 ## What is HyperFormula?
 
-HyperFormula is an open-source calculation engine that allows you
-to perform Excel-like calculations in your business applications.
-It is written in TypeScript and works with major
-JavaScript frameworks like [Vue](/guide/integration-with-vue.md),
-[React](/guide/integration-with-react.md), and [Angular](/guide/integration-with-angular.md). You can use it in a
-browser or as a service, with Node.js as your back-end technology.
+HyperFormula is a headless spreadsheet built on top of TypeScript. It serves as a parser and evaluator of Excel formulas for web applications. You can use it in a browser or as a service, with Node.js as your back-end technology.
```

Comment:
It serves as a parser and evaluator of Excel formulas for web applications.

Better:
It is a parser and evaluator of Excel formulas for web applications.

Reply:
This section is copied from the newly-crafted readme, so I'll change it in both documents.

Reply:
done